.eslintrc.js

```diff
@@ -4,7 +4,10 @@ module.exports = {
         "commonjs": true,
         "es6": true,
         "node": true
-    },
+	},
+	"parserOptions": {
+		"ecmaVersion": 2017,
```

Comment:
Just change [this function](https://github.com/MithrilJS/mithril.js/blob/next/ospec/tests/test-ospec.js#L191) to return what's being awaited. That would be better than this.